andela/bad_ass-ah-backend
# #165082475 user login

**What does this PR do?**

This PR creates login functionality

**Description of Task to be completed?**

- create `POST: /api/users/login` endpoint
- create tests for `POST: /api/users/login` endpoint

**How should this be manually tested?**

- go to postman and type `http://localhost:3000/api/users/login` endpoint
- add the email and password, then send the request
- you should see a successful message with the user info

**What are the relevant pivotal tracker stories?**

[#165082475](https://www.pivotaltracker.com/n/projects/2316906)

**Screenshots (if appropriate)**

You should get a response like this

<img width="1152" alt="Screen Shot 2019-04-03 at 16 38 11" src="https://user-images.githubusercontent.com/32579116/55487775-08a51900-562f-11e9-9fdf-c41db3198c36.png">
